tests/Pagination/PaginationPaginatorTest.php

```diff
@@ -174,4 +174,11 @@ public function testSimplePaginatorReturnsRelevantContextInformation()
             'prev_page_url' => '/?page=1', 'from' => 3, 'to' => 4, 'data' => ['item3', 'item4'],
         ], $p->toArray());
     }
+
+    public function testPaginatorGeneratesUrlsWithoutTrailingSlash()
```

Comment:
Should also ensure that it works on the base root too (e.g. `http://website.com/`) as that *should* end in a slash, right? I know very few people have pagination on their home page, but something like Tumblr would.